packages/mobile/src/screens/track-screen/TrackScreen.tsx

```diff
@@ -95,7 +95,7 @@ const TrackScreenMainContent = ({
           track={track}
           user={user}
           uid={lineup?.entries?.[0]?.uid}
-          isLineupLoading={lineup.isMetadataLoading}
+          isLineupLoading={!lineup?.entries?.[0]}
```

Comment:
Was there a benefit to doing it this way? would this cause an issue if the lineup is loaded, but empty?

Reply:
I changed this because now we load the hero track first (separately from the rest of the lineup which is the "more by this artist") and put that in the lineup immediately, and then make the call to load the rest of the lineup which sets `isMetadataLoading` to true. I.e. if the track has loaded but the rest of the lineup has not, `lineup.isMetadataLoading` will still be false even though we're ready for the user to hit play.

`would this cause an issue if the lineup is loaded, but empty?` - in this case i believe you'd just get an error saying the track is unavailable

This matches the logic in web as well

Reply:
yup, I read the rest of the PR and saw that this was fine 👌